client-common/src/storage.rs

```diff
@@ -60,10 +61,29 @@ pub trait Storage: Send + Sync {
 
     /// Returns all the keyspaces currently available.
     fn keyspaces(&self) -> Result<Vec<Vec<u8>>>;
+
+    /// load and deserialize object
+    fn load<T: Decode>(&self, keyspace: &str, key: &str) -> Result<Option<T>> {
```

Comment:
`Storage` and `SecureStorage` traits are supposed to be agnostic of de/serialisation. If you need a function to automatically de/serialise using `parity_scale_codec`, you can create that function outside of this trait which internally uses `get()` and `set()`.

This applies to all the new functions in these traits which are specific to `parity_scale_codec`.

Reply:
Ok, I can move them to service mod.

Reply:
May I argue for this change a bit more ;D, because it's really convenient this way, and I think we can have a "default" serialization choice for `Storage`, so `parity_scale_codec` is not some arbitrary format, but the default format we use for `Storage`.

Reply:
Ok.